contrib/incident-response/infrastructure/modules/turbinia/templates/scripts/install-turbinia-worker.sh.tpl

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Copyright 2019 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit on any error.
+set -e
+
+# Default constants.
+readonly BOOT_FINISHED_FILE="/var/lib/cloud/instance/boot-finished"
+readonly STARTUP_FINISHED_FILE="/var/lib/cloud/instance/startup-script-finished"
+
+# Redirect stdout and stderr to logfile.
+exec > /var/log/terraform_provision.log
+exec 2>&1
+
+# Exit if the startup script has already been executed successfully.
+if [[ -f "$${STARTUP_FINISHED_FILE}" ]]; then
+  exit 0
+fi
+
+# Wait for cloud-init to finish all tasks.
+until [[ -f "$${BOOT_FINISHED_FILE}" ]]; do
+  sleep 1
+done
+
+# --- MAIN ---
+apt-get update
+apt-get -y install python-pip
+
+# Install Turbinia
+pip install https://github.com/google/turbinia/archive/master.zip
+
+# Turbinia needs a recent version of urllib3
+pip install urllib3 --upgrade
+pip install cryptography --upgrade
+
+# Install Plaso
+add-apt-repository -y ppa:gift/stable
+apt-get -y install python-plaso plaso-tools
+
+# Install Hindsight
+pip install pyhindsight
```

Comment:
I'm a little conflicted about installing these manually instead of using the `extras_require` target for `[worker]`.  I think it's probably better to use Plaso from the Gift PPA, but now we need to manually keep the worker dependencies in sync.  Maybe we should just add a comment in the `turbinia/setup.py` to reference updating this file when any new dependencies are added.

I think this is good for now, but let's chat about how to handle this.

BTW, this will only work under python2 for now because there isn't a python3 version of hindsight yet.  See https://github.com/google/turbinia/issues/413.